clients/fides-js/src/lib/gpp/constants.ts

```diff
@@ -0,0 +1 @@
+export const SUPPORTED_APIS = ["2:tcfeuv2"];
```

Comment:
this will eventually come from the backend, I think

Reply:
from which endpoint exactly?

Reply:
I think it'll eventually come through the privacy experience endpoint. the backend will start recording it in https://ethyca.atlassian.net/browse/PROD-1361